docs/quickstart.rst

```diff
@@ -20,45 +20,45 @@ ASLPrep installation
 
-1. Installation through Docker/ Singularity (**recommended**)
+1. Installation through Docker or Singularity (**recommended**)
 
 For every new version of ``ASLPrep`` that is released, a corresponding Docker image is generated and pushed to `DockerHub <https://hub.docker.com/r/pennlinc/aslprep>`_. In order to run *ASLPrep* Docker images, the Docker Engine must be `installed <https://docs.docker.com/engine/install/>`_.
 
-We recommend usind Docker. The docker image can be pulled from the ASLPrep DockerHub using the command line:: 
+We recommend using Docker or Singularity to run ASLPrep. The docker image can be pulled from the ASLPrep DockerHub using the command line::
 
-    docker pull pennlinc/aslprep:latest    
+    docker pull pennlinc/aslprep:latest
 
-To use singularity,  a singularity image must be installed directly on the system
+To use singularity, a singularity image must be installed directly on the system
 using the following command::
 
     singularity build aslprep.sif docker://pennlinc/aslprep:latest
 
 This requires installation of Singularity version >= 2.5
 
-See  :ref:`run_docker`  and  :ref:`run_singularity` for more information. 
+See  :ref:`run_docker`  and  :ref:`run_singularity` for more information.
 
 2. Installation is available via ``pip``::
 
     python -m pip install aslprep
-    
-This method is not recommended, because it requires external dependencies to be installed. 
+
+This method is not recommended, because it requires external dependencies to be installed.
 
 Running ASLPrep
 ----------------
 
-Running ``ASLPrep`` will require a freesurfer license file (you do not actually need Freesurfer, though), which can be requested `here <https://surfer.nmr.mgh.harvard.edu/registration.htm>`_. Move this license to a folder in your ``$HOME`` directory (to find this location out, run this in the terminal: echo $HOME) called **license**.
+Running ``ASLPrep`` will require a freesurfer license file (you do not actually need Freesurfer, though), which can be requested `here <https://surfer.nmr.mgh.harvard.edu/registration.html>`_. Move this license to a folder in your ``$HOME`` directory (to find the path to your home directory in the terminal, echo $HOME) and call it **license**.
 
 In the Docker desktop application, please select Preferences > Resources > Advanced and select at least 12GB for RAM. Restart Docker.
 
 Move the data directory to your ``$HOME`` directory (again, to find this location out, run this in the terminal: echo $HOME). Make sure it is called **ds000240**.
 
 The following command, which should run in about 8 hours, can be called for a single participant::
 
-    docker run -ti -m 12GB --rm -v $HOME/license.txt:/license/license.txt -v $HOME/ds000240:/data:ro -v $HOME/ds000240-results:/out:rw -v                      $HOME/tmp/ds000240-workdir:/work:rw pennlinc/aslprep /data /out participant --participant-label 01 --fs-license-file /license/license.txt -w /work
```

Comment:
do you also want to include a singularity run example (might not be necessary just a thought)

Reply:
that will be a good option